README.rst

```diff
@@ -227,10 +216,10 @@ client to launch this analysis example, please read on.
 
 We start by creating a `reana.yaml <reana.yaml>`_ file describing the above
 analysis structure with its inputs, code, runtime environment, computational
-workflow steps and expected outputs. In this example we are using the CWL
-workflow specification, with its steps in the `workflow <workflow>`_ directory.
+workflow steps and expected outputs. In this example we are using the Snakemake
+workflow specification, which you can find in the `workflow <workflow>`_ directory.
 
-At the same time, for easy debugging purposes (to avoid the *cwl* overhead), we
+At the same time, for easy debugging purposes, we
```

Comment:
We can remove the whole "easy debugging purposes" paragraph, not to mention any Serial here, and keep only Snakemake mentioned everywhere.